tests/test_regrid2.py

```diff
@@ -1,6 +1,6 @@
 
 import cdms2, numpy.ma, regrid2 as regrid, os, sys
-from regrid2 import Regridder
+from regrid2 import Horizontal
```

Comment:
Because of this warning.
https://github.com/UV-CDAT/cdms/blob/master/regrid2/Lib/horizontal.py#L291-L294